lib/Yield/PTML.cpp

```diff
@@ -535,19 +543,71 @@ static std::string taggedText(const PTMLBuilder &ThePTMLBuilder,
   return Result;
 }
 
+static std::string instructionPrefix(const PTMLBuilder &ThePTMLBuilder,
+                                     const yield::Instruction &Instruction) {
+  std::string Result;
+
+  if (options::InstructionAddressColumnWidth) {
+    std::string SerializedAddress = Instruction.Address().toString();
+    Result += ThePTMLBuilder.getTag(tags::Span, SerializedAddress)
+                .addAttribute(attributes::Token, tokenTypes::InstructionAddress)
+                .serialize()
+              + ' ';
+
+    if (SerializedAddress.size() < options::InstructionAddressColumnWidth) {
+      uint64_t Column = options::InstructionAddressColumnWidth;
+      Result += ThePTMLBuilder
+                  .getTag(tags::Span,
+                          std::string(Column - SerializedAddress.size(), ' '))
+                  .addAttribute(attributes::Token, ptml::tokens::Indentation)
+                  .serialize();
+    } else {
+      // TODO: handle column overflows more carefully.
+    }
+  }
+
+  if (options::RawByteColumnWidth) {
+    std::string SerializedBytes;
+    for (uint8_t Byte : Instruction.RawBytes()) {
+      std::string HexByte = Byte ? llvm::utohexstr(Byte, true, 2) : "00";
```

Comment:
Why is `Byte ? ... : "00"` necessary?

Reply:
`llvm::utohexstr` was printing `0` for some reason for 0s. So instead of trying to figure out what's wrong with it, I just manually handled the problem case and called it a day.